test/e2e/config.go

```diff
@@ -1,7 +1,7 @@
 package integration
 
 var (
-	redis             = "quay.io/libpod/redis:alpine"
+	REDIS_IMAGE       = "quay.io/libpod/redis:alpine"
 	fedoraMinimal     = "registry.fedoraproject.org/fedora-minimal:34"
```

Comment:
I left some untouched (see also below). I lean toward having those be UPCASE also, but those seem unambiguous enough, and at least they have one Capital.